### PR TITLE
Add instances of  `MessagePack` for basic types

### PR DESCRIPTION
**Pull request by Martoon-00**

### CODE REVIEW

File: src/Data/MessagePack/Instances.hs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+module Data.MessagePack.Instances () where
+
+import           Data.MessagePack.Class   (MessagePack (..))
+import           Data.MessagePack.Generic
```
Comment (iphydf) [inline src/Data/MessagePack/Instances.hs]: Add `()` at the end, since we're only importing instances, no other symbols.

**Review by iphydf — CHANGES_REQUESTED** [review]

File: src/Data/MessagePack/Instances.hs (hunk 1 above)
Comment (iphydf) [reply src/Data/MessagePack/Instances.hs]: This comment was on line 5, meaning the `Generic` import. But you're right, `MessagePack` also doesn't need functions. Write this:

``` haskell
import           Data.MessagePack.Class   (MessagePack)
import           Data.MessagePack.Generic ()
```

**Comment by iphydf** [comment]
After that's done, please squash your commits into one.

File: src/Data/MessagePack/Instances.hs (hunk 1 above)
Comment (Martoon-00) [reply src/Data/MessagePack/Instances.hs]: Oh, really. Thank you

File: src/Data/MessagePack/Instances.hs [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+module Data.MessagePack.Instances () where
+
+import           Data.MessagePack.Class   (MessagePack ())
```
Comment (iphydf) [inline src/Data/MessagePack/Instances.hs]: I don't think the () is necessary here. Did you try without it?

**Comment by iphydf** [comment]
Thanks, I don't get notified if there is a change. Perhaps explicitly reply to the review thread when you fix something next time :).

**Review by iphydf — APPROVED** [review]

**Comment by Martoon-00** [comment]
Got it, thanks for your patience!

**Pull request merged**